media/transcoder/create_job_with_periodic_images_spritesheet.go

```diff
@@ -103,8 +104,8 @@ func createJobWithPeriodicImagesSpritesheet(w io.Writer, projectID string, locat
 			},
 		},
 	}
-	// Creates the job, Jobs take a variable amount of time to run.
-	// You can query for the job state.
+	// Creates the job. Jobs take a variable amount of time to run.
+	// You can query for the job state; see getJob() in get_job.go.
```

Comment:
If the user is viewing this sample on cloud.google.com, they won't immediately know where to find `get_job.go`. I was thinking more like giving the name of the client function they could use. A small nudge in the right direction.

Reply:
Added link to this doc, which includes the client function.